package.json

```
@@ -3,7 +3,8 @@
   "version": "0.3.4",
   "description": "RSK JavaScript Wallet API repository",
   "engines": {
-    "node": ">=8.6.0"
+    "node": "10.13.0",
+    "npm": "6.9.0"
```

```suggestion
    "node": ">= 10.13.0",
    "npm": ">= 6.9.0"
```

so that it will accept higher versions as well - I'm assuming that we don't want to stop users from using nodejs v12, etc

---

Yes Brendan, v10, v11, v12 will work.